streampark-console/streampark-console-webapp/src/views/flink/home/components/Modal.vue

```diff
@@ -100,18 +100,20 @@
       flinkHome: formValue.flinkHome,
     });
     const checkResp = parseInt(resp.data);
-    if (checkResp != 0) {
-      // Environment detection is successful
-      if (checkResp == -1) {
-        Swal.fire('Failed', 'FLINK_HOME invalid path.', 'error');
-      } else if (checkResp == 1) {
-        Swal.fire('Failed', t('setting.flinkHome.operateMessage.flinkNameIsUnique'), 'error');
-      } else if (checkResp == 2) {
-        Swal.fire(
-          'Failed',
-          'can no found flink-dist or found multiple flink-dist, FLINK_HOME error.',
-          'error',
-        );
+    if (checkResp !== FlinkEvnEnum.FEASIBLE) {
+      switch (checkResp) {
+        case FlinkEvnEnum.INVALID:
+          Swal.fire('Failed', 'FLINK_HOME invalid path.', 'error');
+          break;
+        case FlinkEvnEnum.NAME_REPEATED:
+          Swal.fire('Failed', t('setting.flinkHome.operateMessage.flinkNameIsUnique'), 'error');
+          break;
+        case FlinkEvnEnum.FLINK_DIST_REPEATED:
+          Swal.fire(
+              'Failed',
+              'can no found flink-dist or found multiple flink-dist, FLINK_HOME error.',
```

Comment:
error msg should use i18n, e.g: t('setting.flinkHome.operateMessage.FLINK_DIST_REPEATED')
...

Reply:
sry I know nothing about the front end